commons-rng-simple/src/main/java/org/apache/commons/rng/simple/ThreadLocalRandomSource.java

```diff
@@ -69,7 +69,7 @@
      *
      * <p>This should only be modified to create new instances in a synchronized block.
      */
-    private static EnumMap<RandomSource, ThreadLocal<UniformRandomProvider>> sources =
+    private static final EnumMap<RandomSource, ThreadLocal<UniformRandomProvider>> sources =
```

Comment:
This now fails checkstyle naming convensions. Please rename to SOURCES.

Reply:
Changed